examples/experimental/i3s-17-and-debug/obb-utils.js

```diff
@@ -0,0 +1,99 @@
+import {Vector3} from '@math.gl/core';
+import {OrientedBoundingBox} from '@math.gl/culling';
+import {Ellipsoid} from '@math.gl/geospatial';
+import {PolygonLayer} from '@deck.gl/layers';
+import {getTileColor} from './coloring-utils';
+
+const LINE_WIDTH = 3;
+const BG_OPACITY = 20;
+
+export function getObbLayer(tile, options) {
+  const data = [
+    {
+      ...getObbBounds(tile.boundingVolume),
+      ...getObbColors(tile, options)
+    }
+  ];
+
+  return new PolygonLayer({
```

Comment:
The thing with OBBs is that they are oriented bounding boxes. Unlike Axis aligned bounding boxes they can have any orientation. 

But the PolygonLayer is a 2.5D layer, it can show flat polygons extruded into the Z dimension (unless I am much mistaken) - so even if you can specify different heights for the base lines, the vertical lines will not be tilted correctly.

It probably works well in many cases. But we might want to create a SimpleMeshLayer with a unit box geometry and make the instances specify orientation scale and sixe.

Reply:
PolygonLayer replaced to SimpleMeshLayer with Cube or Sphere geometry.